src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
 // write code here
+
+const tableElement = document.querySelector('table');
```

Comment:
Ensure that `tableElement` is not null before proceeding. If the table is not found, this will cause an error.